packages/jest-cli/src/snapshot_interactive_mode.js

```diff
@@ -69,12 +81,92 @@ export default class SnapshotInteractiveMode {
     this._pipe.write(messages.filter(Boolean).join('\n') + '\n');
   }
 
+  _drawUIDoneWithSkipped() {
+    this._pipe.write(ansiEscapes.clearScreen);
+    const numPass = this._countPaths - this._testAssertions.length;
+
+    let stats = chalk.bold.dim(
+      pluralize('snapshot', this._countPaths) + ' reviewed',
+    );
+    if (numPass) {
+      stats +=
+        ', ' + chalk.bold.green(pluralize('snapshot', numPass) + ' updated');
+    }
+    if (this._skippedNum) {
+      stats +=
+        ', ' +
+        chalk.bold.yellow(pluralize('snapshot', this._skippedNum) + ' skipped');
+    }
+    const messages = [
+      '\n' + chalk.bold('Interactive Snapshot Result'),
+      ' \u203A ' + stats,
+      '\n' + chalk.bold('Watch Usage'),
+
+      chalk.dim(' \u203A Press ') +
+        'r' +
+        chalk.dim(' to restart Interactive Snapshot Mode.'),
+
+      chalk.dim(' \u203A Press ') +
+        'q' +
+        chalk.dim(' to quit Interactive Snapshot Mode.'),
+    ];
+
+    this._pipe.write(messages.filter(Boolean).join('\n') + '\n');
+  }
+
+  _drawUIDone() {
+    this._pipe.write(ansiEscapes.clearScreen);
+    const numPass = this._countPaths - this._testAssertions.length;
+
+    let stats = chalk.bold.dim(
+      pluralize('snapshot', this._countPaths) + ' reviewed',
+    );
+    if (numPass) {
+      stats +=
+        ', ' + chalk.bold.green(pluralize('snapshot', numPass) + ' updated');
+    }
+    const messages = [
+      '\n' + chalk.bold('Interactive Snapshot Result'),
+      ' \u203A ' + stats,
+      '\n' + chalk.bold('Watch Usage'),
+
+      chalk.dim(' \u203A Press ') +
+        'Enter' +
+        chalk.dim(' to return to watch mode.'),
+    ];
+
+    this._pipe.write(messages.filter(Boolean).join('\n') + '\n');
+  }
+
+  _drawUIOverlay() {
+    this._pipe.write(ansiEscapes.cursorUp(6));
```

Comment:
what's this magical number?

Reply:
I have no idea, @genintho do you know?

Reply:
I remember having trouble to draw the "results" in the position I wanted.
When this function is called, some other lines where already been drawn (I think the instructions, but not sure) and this was a hack to insert characters "before" them.

Reply:
Also the original UI I build got modified with the different review and this might be not necessary anymore.

Reply:
Thanks @genintho, I'll remove and test 👌

Reply:
Ah, got it. It hides these lines:

Reply:
Ah, got it. Hit hides these lines